test/unit/uk/gov/hmrc/individualsemploymentsapi/controller/EmploymentsControllerSpec.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.uk.gov.hmrc.individualsemploymentsapi.controller
+
+import java.util.UUID
+
+import org.scalatestplus.play._
+import play.api.http.Status.OK
+import play.api.libs.json.Json.parse
+import play.api.mvc._
+import play.api.test.Helpers._
+import play.api.test._
+import uk.gov.hmrc.domain.Nino
+import uk.gov.hmrc.individualsemploymentsapi.controller.EmploymentsController
+import uk.gov.hmrc.individualsemploymentsapi.error.ErrorResponses.MatchNotFoundException
+
+import scala.concurrent.Future.{failed, successful}
+
+class EmploymentsControllerSpec extends PlaySpec with Results {
```

Comment:
I think you can remove this class as it is covered in SandboxEmploymentsControllerSpec and class should not be instantiated if abstract.